### PR TITLE
Add Visist table

### PR DESCRIPTION
**Pull request by farouqdaif1**
Created tables vets and "join table" specialization and visits with many-to-many relationship 
Inserted data are given in the requirement 
queries to answer the questions

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -107,4 +107,38 @@ UPDATE animals SET owner_id=3 WHERE name IN ('Devimon','Plantmon');
...
+
+
+/*Insert the following data for vets:
+Vet William Tatcher is 45 years old and graduated Apr 23rd, 2000.
+Vet Maisy Smith is 26 years old and graduated Jan 17th, 2019.
+Vet Stephanie Mendez is 64 years old and graduated May 4th, 1981.
+Vet Jack Harkness is 38 years old and graduated Jun 8th, 2008.
+*/
+INSERT INTO vets (full_name,age,date_of_graduation )
+VALUES('William Tatche', 45,'2000-04-23'),
```
Comment (SajjadAhmad14) [inline data.sql:122]: Please correct the type here. It should be ``William Tatcher``.

File: data.sql [hunk 2]
```diff
@@ -107,4 +107,38 @@ UPDATE animals SET owner_id=3 WHERE name IN ('Devimon','Plantmon');
...
+(' Maisy Smith', 26, '2019-01-17'),
+('Stephanie Mendez',64,'1981-05-04'),
+('Jack Harkness', 38,'2008-06-08');
```
Comment (SajjadAhmad14) [inline data.sql:125]: Please correct the type here. It should be `Jack Harknes`.

File: data.sql [hunk 3]
```diff
@@ -107,4 +107,38 @@ UPDATE animals SET owner_id=3 WHERE name IN ('Devimon','Plantmon');
 --Melody Pond owns Charmander, Squirtle, and Blossom.
 UPDATE animals SET owner_id=4 WHERE name IN ('Squirtle','Blossom','Charmander');
```
Comment (SajjadAhmad14) [inline data.sql:108]: Please add the project title and description in README. This way it will look more professional and descriptive for the end user.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Farouq, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
PR has a title and description. :heavy_check_mark:
Git flow is followed. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
**_Note: Please do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by descholar-ceo — APPROVED** [review]
Hi @farouqdaif1 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**